lib/src/base-styles/_nx-page-header.scss

```diff
@@ -17,7 +17,7 @@
   box-sizing: content-box;
 
   background-color: var(--nx-swatch-white);
-  border-bottom: 1px solid var(--nx-swatch-grey-90);
+  border-bottom: 1px solid var(--nx-swatch-indigo-90);
```

Comment:
Just wondering if this should use `--nx-color-border`

Reply:
> Just wondering if this should use `--nx-color-border`

Thanks for catching that! [ad970fa](https://github.com/sonatype/sonatype-react-shared-components/pull/980/commits/ad970faf3b34c45366b877f1c9378b4370cfabcb)